### PR TITLE
Issue464 move io html

### PR DESCRIPTION
**Pull request by EttoreZ**
This pull request is for issue #464

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -58,3 +58,9 @@ object in the process of initializing the test.
 data only for the requested index or period of time. This
 method would be used by the ``KPI_Calculator`` or
 ``Forecaster`` modules.
+
+## Testcase I/O generation for documentation
```
Comment (dhblum) [inline data/README.md:62]: Suggest "Generate Documentation for Test Case I/O Points"

File: data/README.md [hunk 2]
```diff
@@ -58,3 +58,9 @@ object in the process of initializing the test.
...
+``get_html_IO.py`` can be used to generate the html documentation code for I/O.
+Outputs:
+"inputs.txt": html code documenting the inputs
```
Comment (dhblum) [inline data/README.md:65]: Make into bullet point list so renders correctly on github page

File: data/README.md [hunk 3]
```diff
@@ -58,3 +58,9 @@ object in the process of initializing the test.
...
+"measurements.txt": html code documenting the outputs
```
Comment (dhblum) [inline data/README.md:66]: Add also some brief instructions on how to use.  For instance that you should first deploy the test case of interest and then run this script with python, maybe just showing the cmd line command.

**Review by dhblum — CHANGES_REQUESTED** [review]
Thanks @EttoreZ.  In addition to my inline comments: 

- Please also add a line to the ``releasenotes.md`` under the heading for a backwards compatible change that does not impact benchmark results.  Add the heading if it does not already exist to the xx/xx/xxxx release section (top) and you can look to previous releases for guidance on what to write and how to format, etc.
- Add your name to ``contributors.md`` if it doesn't exist already.

**Comment by dhblum** [comment]
@EttoreZ just for your information I merged https://github.com/ibpsa/project1-boptest/pull/467.

**Comment by EttoreZ** [comment]
> @EttoreZ just for your information I merged #467.

Thank you for the heads up @dhblum. I updated the pull request with the suggested changes.

File: data/README.md [hunk 4]
```diff
@@ -59,8 +59,12 @@ data only for the requested index or period of time. This
...
-"inputs.txt": html code documenting the inputs
-"measurements.txt": html code documenting the outputs
\ No newline at end of file
+## Generate Documentation for Test Case I/O Points
+``get_html_IO.py`` can be used to generate the html documentation code for I/O in alphabetical order.
+Outputs of the script:
+- ``inputs.txt``: html code documenting the inputs
+- ``measurements.txt``: html code documenting the outputs
+In order to run the script deploy the BOPTEST testcase by using the command 
+``$ TESTCASE=<testcase_name> docker-compose up``. Then open a terminal in
```
Comment (dhblum) [inline data/README.md:68]: @EttoreZ Just documenting what we discussed, please remove this command for deploying test case and refer to root readme so as not to duplicate.

**Review by dhblum — CHANGES_REQUESTED** [review]
Thanks @EttoreZ for the updates.  Please just address the one additional inline comment and then I think this will be ready to go.

**Comment by EttoreZ** [comment]
Thanks for the careful review @dhblum. I addressed the last comments and updated the PR

**Pull request merged**